java/src/jmri/implementation/AbstractTurnout.java

```diff
@@ -186,9 +188,9 @@ protected void newKnownState(int s) {
     }
 
     /**
-     * Show whether state is one you can safely run trains over
+     * Show whether state is one you can safely run trains over.
      *
-     * @return    true iff state is a valid one and the known state is the same as
+     * @return true if state is a valid one and the known state is the same as
```

Comment:
"iff" is likely not a spelling error. See https://en.wikipedia.org/wiki/If_and_only_if